### PR TITLE
feature/WAL-686

### PR DESCRIPTION
**Pull request by BtencateSphereon**
PR requires updated ssi-sdk packages

### CODE REVIEW

File: src/screens/SSIContactAddScreen/index.tsx [hunk 1]
```diff
@@ -110,7 +94,7 @@ class SSIContactAddScreen extends PureComponent<IProps, IState> {
       filter: [
         {
           contact: {
-            displayName: name,
+            legalName: name,
```
Comment (nklomp) [inline src/screens/SSIContactAddScreen/index.tsx:97]: Is this correct?

Comment (BtencateSphereon) [reply src/screens/SSIContactAddScreen/index.tsx:97]: displayName is not unique and currently we only support organizations

Comment (nklomp) [reply src/screens/SSIContactAddScreen/index.tsx:97]: Okay suggest to add a comment about that then

**Review by nklomp — COMMENTED** [review]
See comment remark. Rest LGTM

File: src/screens/SSIContactAddScreen/index.tsx (hunk 1 above)
Comment (BtencateSphereon) [reply src/screens/SSIContactAddScreen/index.tsx:98]: added a comment

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**